packages/mds-policy-service/service/provider.ts

```diff
@@ -83,7 +86,7 @@ export const PolicyServiceProvider: ServiceProvider<PolicyService, PolicyService
         throw new DependencyMissingError(`some geographies not published!`)
 
       const publishedPolicy = await PolicyRepository.publishPolicy(policy_id, publish_date, {
-        beforeCommit: async policy => PolicyStreamKafka.write(policy)
+        beforeCommit: async policy => (process.env.KAFKA_HOST ? PolicyStreamKafka.write(policy) : undefined)
```

Comment:
Could we make the callback function itself undefined, so we don't call anything if we don't need to?

Reply:
This wouldn't actually change anything, because `publishPolicy` defaults the callback to being `async () => undefined` if `beforeCommit` is undefined anyways.

Reply:
Resolved in df6be78